active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,320 @@
+
+## API Notes
+
+### Notable Properties  
+
+| Name | Description |
+|:-:|:--|
+| StepFrequency | Gets or sets the value part of a value range that steps should be created for. |
+
+## API Details
+<!-- todo: Missing some APIs in here, like TemplatSettings class and event args classes -->
+
+```c++ 
+enum NumberBoxSpinButtonPlacementMode
+{
+    Hidden,
+    Compact,
+    Inline,
+};
+
+enum NumberBoxValidationMode
+{
+    InvalidInputOverwritten,
+    Disabled,
+};
+
+runtimeclass NumberBoxValueChangedEventArgs
+{
+    Double OldValue{ get; };
+    Double NewValue{ get; };
+};
+
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
+{
+    NumberBox();
+
+    Double Minimum;
+    Double Maximum;
+    Double Value;
+    Double StepFrequency;
+
+    String Header;
+    DataTemplate HeaderTemplate;
+    String Text; 
+    String PlaceholderText;
+    
+    FlyoutBase SelectionFlyout;
+    SolidColorBrush SelectionHighlightColor;
+    TextReadingOrder TextReadingOrder;
+    Boolean PreventKeyboardDisplayOnProgrammaticFocus;
+    Object Description;
+  
+    NumberBoxValidationMode ValidationMode;
+
+    Boolean AcceptsCalculation;
+
+    NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
+
+    Boolean IsHyperScrollEnabled;
+    Boolean IsWrapEnabled;
+
+    Windows.Globalization.NumberFormatting.INumberFormatter2 NumberFormatter;
+
+    event Windows.Foundation.TypedEventHandler<NumberBox, NumberBoxValueChangedEventArgs> ValueChanged;
+    
+    static Windows.UI.Xaml.DependencyProperty MinimumProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaximumProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ValueProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty StepFrequencyProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty Header{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplate{ get; };
+    static Windows.UI.Xaml.DependencyProperty Text{ get; };
+    static Windows.UI.Xaml.DependencyProperty PlaceholderText{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty ValidationModeProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty AcceptsCalculationProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty SpinButtonPlacementModeProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty IsHyperScrollEnabledProperty{ get; };   
+    static Windows.UI.Xaml.DependencyProperty IsWrapEnabled{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty NumberFormatter{ get; };
+}
+```
+
+## Appendix
+
+### Behavioral Components
```

Comment:
Recommend: Some of this looks more important than the Appendix, should be in DMC

Reply:
Discussed: Already surfaced in Examples and Remarks.